src/core/Akka.Persistence.Query/Properties/AssemblyInfo.cs

```diff
@@ -1,17 +1,20 @@
-﻿using System.Reflection;
-using System.Runtime.CompilerServices;
+﻿//-----------------------------------------------------------------------
+// <copyright file="AssemblyInfo.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
```

Comment:
None of the copyright notices have been updated, I assume they would all be updated in one shot?